src/app/components/SelectorButton/SelectorButton.test.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import { mount } from "enzyme"
+import SelectorButton from "./SelectorButton"
+
+beforeEach(() => {
+  jest.resetModules()
+})
+
+const getSelectorButtonWithLockedContext = (context = {}) => {
+  jest.doMock("../../contexts", () => {
+    return {
+      LockContext: {
+        Consumer: props => props.children(context)
+      }
+    }
+  })
+  require("./SelectorButton").default
```

Comment:
```suggestion

```

Why is this `require` required?

Reply:
Essentially it re-imports the selector button with context applied. This is hacky context/jest stuff, but it's the solution used by most people right now.